### DISCUSSION TITLE
Crazy energy usage? must be wrong

### DISCUSSION THREAD

**Discussion by datsun88** [UtilitechAS/amsreader-firmware#467] [General]
Today the data from my amsreader seems to have become broken for me. It says that I've used 4294900kWh one hour and nothing for the rest of the day. I am on version v2.2.5.

Is there a way to reset this? I saw an earlier post that was similar, but I'm pretty new to this so not really sure how to fix.
MQTT data seems to be wrong too

Heres are screenshots:
![screenshot 319](https://user-images.githubusercontent.com/123263308/221638237-4da229ba-854b-450e-941c-fe4e75f9417a.jpg)
![screenshot 320](https://user-images.githubusercontent.com/123263308/221646378-1ae4ada5-d0cb-4e22-b714-54fcbbacf40b.jpg)

**Comment by ArnieO** [comment]
Ouch... data corruption - for some reason. We've seen 2-3 such cases lately.

Please see this blogpost for instructions on how to edit the data, hope it solves your issue: https://amsleser.no/blog/post/28-correcting-stored-values-by-editing-the-configuration-file

**Comment by gskjold** [comment]
This should be fixed from v2.2.12 and later
